defaults/jq/semconv.jq

```diff
@@ -104,3 +108,51 @@ def semconv_grouped_metrics($options): semconv_metrics($options) | semconv_group
 
 # Convenience function to group all metrics by their root namespace without any filtering options.
 def semconv_grouped_metrics: semconv_grouped_metrics({});
+
+#################
+# Event Functions
+#################
+
+# Extracts and processes semantic convention events based on provided options.
+# $options is an object that can contain:
+# - exclude_stability: a list of stability statuses to exclude.
+# - exclude_deprecated: a boolean to exclude deprecated events.
+# - exclude_event_namespace: a list of event domain to exclude.
+def semconv_events($options):
```

Comment:
I'd suggest we hold off on providing this function in weaver for now.  We can directly encode this in semconv repo for markdown generation while we work on the data model to clean up issues around events and make hard-than-this-PR decisions around, e.g. namespacing and group-level deprecation.

Reply:
Let me move this to the required "test" `weaver.yaml` definitions then. Will be tomorrow (more likely Friday) before I get to this, so any additional comments by others (to roll into a single update) would be appreciated.

Reply:
All changes in this file have been reverted